app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -167,24 +173,35 @@ class BrowserFragment : Fragment(), BackHandler {
 
         startPostponedEnterTransition()
 
-        QuickActionComponent(
+        quickActionSheetStore = Store(
+            QuickActionSheetState(
+                readable = getSessionById()?.readerable ?: false,
+                bookmarked = findBookmarkedURL(getSessionById()),
+                readerActive = getSessionById()?.readerMode ?: false,
+                bounceNeeded = false
+            ),
+            ::quickActionSheetStateReducer
```

Comment:
We changed `SearchStore` today to use a class instead of a typealias. This makes it easier to read and it solved a problem where the method reference captured scope and caused a leak. Let's do that here as well to make sure? https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/java/org/mozilla/fenix/search/SearchStore.kt#L16

Reply:
Done :)